src/SafeAreaView.web.tsx

```diff
@@ -19,67 +23,66 @@ const edgeBitmaskMap: Record<Edge, number> = {
   left: LEFT,
 };
 
-export function SafeAreaView({
-  style = {},
-  mode,
-  edges,
-  ...rest
-}: NativeSafeAreaViewProps) {
-  const insets = useSafeAreaInsets();
+export const SafeAreaView: React.ForwardRefExoticComponent<
+  NativeSafeAreaViewProps & React.RefAttributes<NativeSafeAreaViewInstance>
+> = React.forwardRef<NativeSafeAreaViewInstance, NativeSafeAreaViewProps>(
+  ({ style = {}, mode, edges, ...rest }, ref) => {
```

Comment:
If we take `props` here and destructure it inside the component body, does it avoid the extra indentation?

Reply:
Not needed anymore as removing the return type was enough to remove the extra indentation.